CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 ## Changelog
 
+#### WIP
+- Added support for passing ecto `dynamic` filters
+- Added support for passing functions to use for custom filtering
```

Comment:
I don't think you should do this, ultimately this is why the first parameter is a query and can be passed in, because you can add custom filtering there